src/components/StoriesGroup/types.ts

```diff
@@ -0,0 +1,7 @@
+import {StoriesItem, StoriesItemMedia} from '../Stories/types';
+
+export interface StoriesGroupItem {
+    items: StoriesItem[];
+    /** Custom image for preview, otherwise first StoriesItem.media from StoriesItem will be used */
+    previewMedia?: StoriesItemMedia;
```

Comment:
There is `selectedFromThumbnail` in https://github.com/gravity-ui/uikit/pull/387/files#diff-d7c93a4851676d09aa02706b71ceadeb04226b246e0050535331bcffae7d3812R23. I think it is better to name it `thumbnailMedia`